src/exo/API.py

```diff
@@ -233,6 +249,9 @@ def get_ast(self, pattern=None):
                 assert len(results) == 1
                 return results[0]
 
+    def body(self):
```

Comment:
rename to `root`?

Reply:
Removed outright since it's untested/unused and pending further API discussion.